docs/scenarios_authoring/Enrichers.md

```diff
@@ -41,6 +41,28 @@ DatabaseLookupEnricher is a specialized look-up component; it returns all column
 
 ![alt_text](img/databaseLookupEnricher.png "databaseLookup Enricher")
 
+### DecisionTable enricher
+
+This component allows the user to match input to a pre-defined result. Please be wary that this component works a bit different than a classical decision table([Decision tables in Wikipedia](https://en.wikipedia.org/wiki/Decision_table)). It consists of the fields:
+- Name: the name of the component.
+- Decision Table: a user defined table of type TabularTypedData, discussed below.
+- Match condition: a Boolean SpEL expression that matches input to table rows, discussed below.
```

Comment:
1. `matches input to table rows` - I don't think it's possible to grasp the idea without checking the example. 
I'd rather say that for each row Nu evaluates the expression and based on the result (true/false) the currently analyzed row will be picked or not.

2. `discussed below.` - IMO we should deeply explain what the params offer. The example is an addition. Not a main part of the description of the component. 

3. Please, write a little bit about the `#ROW` variable. Explaining what's this.

Reply:
1. I would rephrase it as `will be used to match input to table rows `. I feel that the phrasing you proposed is too complex.

2. I don't think it is a good idea to try and explain these two (decision table, match condition) complex fields in a quick summary. That is precisely why i have extracted them. 

3. Sure, I forgot about that

Reply:
2. I'd insist on splitting the doc into two parts: the explanation and the example. The example is just an addition. An example should not be used as an explanation for a concept.

Reply:
I have added a quick field summary at the top. I don't agree with your " The example is just an addition. ". This is a pretty straightforward, well-known case of functionality. We should present how it works in our software. An example is the perfect way. I also don't see how we can do without it? If the current version finds out to be too little, please argument your idea